django_lti_tool_provider/tests/test_views.py

```diff
@@ -170,10 +162,40 @@ def test_given_session_and_lti_uses_lti(self, patched_send_lti_received):
 
         response = self.send_lti_request(self.get_correct_lti_payload())
 
-        self._verify_lti_created(self.user, self._data)
+        # Should have been created.
+        user = User.objects.all()[0]
+        self._verify_lti_created(user, self._data)
         self._verify_redirected_to(response, self._url_base + self.DEFAULT_REDIRECT)
-        self._verify_lti_updated_signal_is_sent(patched_send_lti_received, self.user)
+        self._verify_lti_updated_signal_is_sent(patched_send_lti_received, user)
 
+    def test_force_login_change(self, patched_send_lti_received):
+        self.assertFalse(User.objects.exclude(id=1))
+        payload = self.get_correct_lti_payload()
+        request = self.send_lti_request(payload, client=RequestFactory())
+        engine = import_module(settings.SESSION_ENGINE)
+        request.session = engine.SessionStore()
+        request.user = None
```

Comment:
@Kelketek you're likely to have good reason to not use `client.session` - could you please tell me so that I'm aware of it too?